pyjobshop/cpoptimizer/create_model.py

```diff
@@ -22,14 +25,19 @@
 )
 
 
-def create_model(data: ProblemData) -> CpoModel:
+def create_model(
+    data: ProblemData,
+    initial_solution: Optional[Solution] = None,
+) -> CpoModel:
     """
     Creates a CP Optimizer model for the given problem data.
 
     Parameters
     ----------
     data
         The problem data instance.
+    initial_solution
```

Comment:
Do we allow incomplete solutions? I think no, because Solutions are now assumed to be complete (see #129).